### PR TITLE
Temporarily disable atttests until a plan is made so that they don't break npm test locally

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

**Comment by m6jones** [comment]
We shouldn't disable accessibility testing. Install attest is the best option. I'll look into the fix right now.

**Comment by jeroach** [comment]
@m6jones 

I don't want to, but this has caused tons of headaches for our team.

It needs to be such that someone can `git clone`  + `npm I` + `npm test` and see no errors

Breaking `npm test` locally and not telling anyone about it is not cool :(
https://github.com/BrightspaceHypermediaComponents/activities/pull/119

**Comment by m6jones** [comment]
I agree - it was our bad for breaking the tests but I will fix it right now.

**Comment by jeroach** [comment]
@m6jones thank you, much appreciated!! :)

**Comment by m6jones** [comment]
Good news: https://github.com/BrightspaceHypermediaComponents/activities/pull/135

**Comment by m6jones** [comment]
I think we should do this PR over mine. Given that we might be switching to `axe-core` and it still doesn't work for everyone.

File: test/index.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
 	<script>
 		// Load and run all tests (.html, .js):
 		WCT.loadSuites([
-			'd2l-activity-list-item/d2l-activity-list-item-test.html?wc-shadydom=true&wc-ce=true',
-			'd2l-activity-list-item/d2l-activity-list-item-test.html?dom=shadow',
+			//'d2l-activity-list-item/d2l-activity-list-item-test.html?wc-shadydom=true&wc-ce=true',
+			//'d2l-activity-list-item/d2l-activity-list-item-test.html?dom=shadow',
```
Comment (m6jones) [inline test/index.html]: You shouldn't need to comment out these if you are deleting the attest-helper.

Comment (jeroach) [reply test/index.html]: I thought that as well, but it broke the `lint` step

Comment (jeroach) [reply test/index.html]: Regardless, I'll reopen and try it out

Comment (m6jones) [reply test/index.html]: okay, I can help make those work. Since we need those tests for our component.

Comment (m6jones) [reply test/index.html]: https://github.com/BrightspaceHypermediaComponents/activities/blob/master/test/d2l-activity-list-item/d2l-activity-list-item-test.js#L175

**Comment by jeroach** [comment]
@m6jones I assume we want to keep the `attest-helper.js` in there? to enable local testing for those that want to verify accessibility stuff

**Comment by jeroach** [comment]
> @m6jones I assume we want to keep the attest-helper.js in there? to enable local testing for those that want to verify accessibility stuff

Or not? Build failed due to trying to install `attest`

**Comment by jeroach** [comment]
Now I'm confused: https://travis-ci.com/BrightspaceHypermediaComponents/activities/builds/105887037

```
2.08s$ npm install attest --registry=https://d2lartifacts.artifactoryonline.com/d2lartifacts/api/npm/attest/
npm ERR! code E404
npm ERR! 404 Not Found - GET https://d2lartifacts.artifactoryonline.com/d2lartifacts/api/npm/attest/depd - not_found
npm ERR! 404 
npm ERR! 404  'depd@^2.0.0' is not in the npm registry.
npm ERR! 404 Your package name is not valid, because 
npm ERR! 404  1. name can only contain URL-friendly characters
npm ERR! 404 It was specified as a dependency of 'attest'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-03-26T17_38_16_040Z-debug.log
The command "npm install attest --registry=https://d2lartifacts.artifactoryonline.com/d2lartifacts/api/npm/attest/" failed and exited with 1 during .
```

**Comment by m6jones** [comment]
You have to update the travis file

**Comment by jeroach** [comment]
And we're green 😄

**Pull request merged**